.travis.yml

```diff
@@ -26,58 +30,69 @@ cache:
   - .stack-work
 
 install:
-  # 1. Download and unpack the stack executable
+  # 1. Download, unpack, and check the stack executable
   - mkdir -p ~/.local/bin
   - export PATH=$HOME/.local/bin:$PATH
-  - travis_retry curl -L https://github.com/commercialhaskell/stack/releases/download/v1.9.3/stack-1.9.3-linux-x86_64.tar.gz | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
-  # 2. Configure stack to use the system GHC installation
-  - stack config set system-ghc --global true
-  - export PATH=/opt/ghc/ghc-8.6.4/bin:$PATH
-  # 3. Init Application Config
+  - travis_retry curl -L https://get.haskellstack.org/stable/linux-x86_64.tar.gz | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
```

Comment:
I would stay with the specifying the concrete version of Stack. When we want to upgrade to a newer one (same as LTS, Pandoc,..), we should do it manually and we should know about the upgrade.

Reply:
Fixed in https://github.com/ds-wizard/dsw-server/pull/61/commits/8f1371210d990a2652c07e43231558be432ace83